keycloak_client.go

```diff
@@ -280,10 +281,10 @@ func (c *Client) delete(accessToken string, plugins ...plugin.Plugin) error {
 				Message:    string(resp.Bytes()),
 			}
 		case resp.StatusCode >= 400:
-			var response map[string]string
+			var response map[string]interface{}
```

Comment:
May be you could factorize the code for StatusCode>=400 which is repeated 3 times